ge/org.osate.ge.gef.ui/src/org/osate/ge/gef/ui/editor/AgeEditor.java

```diff
@@ -465,10 +466,21 @@ public void afterModelChangeNotification() {
 	private final DiagramModificationListener diagramModificationListener = new DiagramModificationAdapter() {
 		@Override
 		public void modificationsCompleted(final ModificationsCompletedEvent e) {
+			final List<?> diagramNodes = selectionProvider.getSelection().toList();
```

Comment:
Should be safe to just force the case to ? extends DiagramNode instead of just using <?>. Checking the type of the contents later should not be necessary.